v3.1/glfw/glfw/src/internal.h

```diff
@@ -56,6 +56,8 @@
  #error "No supported client library selected"
 #endif
 
+#define NS_FORMAT_ARGUMENT(A)
```

Comment:
Per my previous comment, this is a file synced with upstream glfw releases. We likely wasn't to add this upstream instead